src/istio/control/http/attributes_builder.cc

```diff
@@ -169,6 +169,11 @@ void AttributesBuilder::ExtractCheckAttributes(CheckData *check_data) {
     }
   }
   builder.AddString(utils::AttributeName::kContextProtocol, protocol);
+
+  builder.AddStringMap(utils::AttributeName::kRequestQueries,
```

Comment:
should we check if query is empty?  if empty, not to add the attribute,

Reply:
Done.